src/tsolvers/egraph/Explainer.cc

```diff
@@ -48,10 +48,10 @@ void Explainer::storeExplanation(ERef x, ERef y, PtAsgn reason)
     // balanced (which is a requirement to keep the O(nlogn) bound
 
     // Make sure that x is the node with the larger number of edges to switch
-    const Enode& root_x = getEnode(getEnode(x).getRoot());
-    const Enode& root_y = getEnode(getEnode(y).getRoot());
+    Enode const & root_x = getEnode(getEnode(x).getRoot());
+    Enode const & root_y = getEnode(getEnode(y).getRoot());
     if ( root_x.getSize() < root_y.getSize() ) {
-        swap(x,y);
+        std::swap(x,y);
```

Comment:
Seeing this now, I got a little worried. I thing we should not create the named references `root_x` and `root_y` and just inline them in the `if` check.
Because if the `swap` does occur, suddenly `root_x` is the root of `y` and `root_y` is the root of `x`, which is super confusing.
Hopefully we are not using these references later...

Reply:
Also, this made me realize that I have inadvertently changed the meaning of this code in #314. Because `getSize()` now returns the number of children and this should now be `getEqSize()`.

Reply:
> Hopefully we are not using these references later...

They're not used afterwards.  I could make a scope just for this to make it clearer?

> Also, this made me realize that I have inadvertently changed the meaning of this code in #314. Because getSize() now returns the number of children and this should now be getEqSize().

I'll fix it in a follow-up commit.  If I understood you correctly, here it's enough to change `getSize()` => `getEqSize()`?

Reply:
Yes, but I would like to measure the impact of this and also check that the other usages are correct.

Reply:
> They're not used afterwards. I could make a scope just for this to make it clearer?

Yes, scope is also fine.